modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/js/diff-view.js

```diff
@@ -1,9 +1,18 @@
-$("#diff-view-version").on('change',function(){
-    var diff_view_url = "/publisher/pages/diff?type=wsdl&path=" + $("#diff-view-version").val() + ',' +
-        $("#diff-view-version").find(':selected').data('base_path');
+$("#diff-view-version li").on('mouseover',function(event){
```

Comment:
Thanks @splinter , I have fixed it in [this commit](https://github.com/tmkasun/product-greg/commit/3a678f754aef8776cbf4f7854bf8588dd14e7730)